packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -15,36 +16,34 @@ import {
 import { getLegalEntityName, isDesktop, isMobile, routes, validPhone } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
 import FormSubHeader from 'Components/form-sub-header';
-import PoiNameDobExample from 'Assets/ic-poi-name-dob-example.svg';
 import InlineNoteWithIcon from 'Components/inline-note-with-icon';
 import FormBodySection from 'Components/form-body-section';
 import { DateOfBirthField, FormInputField } from 'Components/forms/form-fields';
-import { Link } from 'react-router-dom';
-import { getEmploymentStatusList } from 'Sections/Assessment/FinancialAssessment/financial-information-list';
 import { isFieldImmutable } from 'Helpers/utils';
+import { getEmploymentStatusList } from 'Sections/Assessment/FinancialAssessment/financial-information-list';
 
 const PersonalDetailsForm = props => {
     const {
+        inline_note_text,
         is_virtual,
```

Comment:
I assume this prop is coming from the client store. I also think youre using this for a condition somewhere 🤔 
We can get this `is_virtual` check from `useActiveAccount`. Lets reduce the number of store dependancy wherever possible slowly. If its too much of a refactor and out of scope, feel free to ignore this comment 😸

Reply:
This card was opened way before the implementation of hooks. It was blocked for a long time. Will incorporate hooks in a separate card as this feature is urgent